src/awx_plugins/credentials/github_app.py

```diff
@@ -132,18 +132,20 @@ class EmptyKwargs(TypedDict):
 
 GH_CLIENT_ID_TRAILER_LENGTH = 16
 HEXADECIMAL_BASE = 16
+GH_CLIENT_ID_MIN_LENGTH = 19
+GH_CLIENT_ID_MAX_LENGTH = 22
```

Comment:
Let's try keeping coupled values together:
```suggestion
GH_CLIENT_ID_LENGTH_BOUNDS = range(19, 23)  # min 19, max 22
```

Reply:
Fixed